apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -120,7 +122,12 @@ const PublishNotes: React.FunctionComponent = () => {
               ? spinnerIconProps
               : {iconName: 'upload'}
           }
-          disabled={isReadOnly || !hasFilledOutModelCard || saveInProgress}
+          disabled={
```

Comment:
There's an edge case when a user fills out an unpublished model card completely and clicks on 'save'. The 'publish' button is also disabled which may be confusing. Could add to `(!havePropertiesChanged && modelCardInfo.isPublished)` to condition.

Reply:
This is probably out of scope of this PR, but when the 'Publish' button is disabled, the 'Ready to publish' notification is still displayed at the top of the panel which may also be a little confusing.
We could remove this notification when the 'Publish' button is disabled - maybe as a follow-up? Currently, we either display the `PublishOkNotification` or the `CompleteToPublishNotification`.

Reply:
Oh good call! Yep let me take another look at this and can file a follow-up if needed.

Reply:
Hmm yeah this is a little confusing - I added your first condition which makes sense for the first publish, but if the user then changes some information and saves, the publish button is disabled. I think it might make sense to always have the Publish button enabled (if all fields are filled out)? Even if there haven't been any updates, we'll skip the save, but we'll still change to the presentation view.

I might merge this is as-is and do a fast follow to implement this behavior since it'll involve some other redux-related changes.